src/Sylius/Bundle/AdminBundle/Resources/config/app/sylius/resources/admin_user.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+use Sylius\Bundle\AdminBundle\Form\Type\AdminUserType;
+use Sylius\Resource\Metadata\BulkDelete;
+use Sylius\Resource\Metadata\Create;
+use Sylius\Resource\Metadata\Delete;
+use Sylius\Resource\Metadata\Index;
+use Sylius\Resource\Metadata\Operations;
+use Sylius\Resource\Metadata\ResourceMetadata;
+use Sylius\Resource\Metadata\Update;
+
+return (new ResourceMetadata())
+    ->withRoutePrefix('/admin')
+    ->withClass('%sylius.model.admin_user.class%')
+    ->withSection('admin')
+    ->withTemplatesDir('@SyliusAdmin\\shared\\crud')
+    ->withFormType(AdminUserType::class)
+    ->withOperations(new Operations([
+        new Create(path: 'users/new', redirectToRoute: 'sylius_admin_admin_user_index'),
+        new Update(path: 'users/{id}/edit', redirectToRoute: 'sylius_admin_admin_user_index'),
+        new Delete(path: 'users/{id}/delete'),
+        new BulkDelete(path: 'users/bulk_delete'),
+        new Index(path: 'users', grid: 'sylius_admin_admin_user'),
```

Comment:
What happens if no `path` is given?
I believe each operation is not aware of the root `ResourceMetadata` making more verbose setting all operations. Does a way to avoid settings the `path` at least exist ?

Reply:
Reading the other resource definitions I see that the `path` is not required so this is here for a good reason right?